manifests/bridges.pp

```diff
@@ -2,12 +2,12 @@
 class nftables::bridges (
   # lint:ignore:parameter_documentation
   Enum['present','absent'] $ensure = 'present',
-  Regexp $bridgenames = /^br.+/
```

Comment:
Is it documented somewhere that you can't use rich data types here ?

Reply:
Lets ask @joshcooper tomorrow during #office-hours